deploy/kiali/kiali_cr.yaml

```diff
@@ -419,10 +419,7 @@ spec:
 #   use_kiali_token: When true and if auth.type is "bearer", the same OAuth token used for authentication in Kiali will be used for the API calls to Grafana,
 #       and auth.token config is ignored then.
 #   username: Username to be used when making requests to Grafana, for basic authentication. User only requires viewer permissions.
-# component_status: Grafana info for the Istio Component Status masthead indicator. Only if it is enabled.
-#   app_label: The value of the Grafana pod(s)'s app label. The name of the app label is defined in the setting 'istio_labels.app_label_name'. It defaults to grafana.
-#   is_core: Whether the component is core for your deployment. It defaults to false.
-#   namespace: The namespace where Grafana is installed in. It defaults to the 'istio_namespace' setting.
+# core_component: Used in the Components health feature. When true, the unhealthy scenarios will be raised as errors. Otherwise, they will be raised as a warning.
```

Comment:
I think I prefer with the `is_` prefix to make it clear that a boolean is expected without having to refer to doc